modules/machines-edge-machine-pool.adoc

```diff
@@ -0,0 +1,103 @@
+
+:_content-type: CONCEPT
+[id="machines-edge-machine-pool_{context}"]
+= The edge machine pool
+
+{product-title} 4.12 introduced a new compute pool, _edge_, that is designed for use in remote zones. The edge compute pool configuration is common between AWS Local Zone locations. However, due to the image type and size limitations on Local Zone resources, the default instance type that is created can vary from the traditional worker pool.
+
+The default elastic block store (EBS) for Local Zone locations is `gp2`, which differs from the default worker pool. The preferred list of instance types for the edge compute pool follows the same order of worker pools. Depending on the availability of the location, the system chooses one of those instances. The preferred list of instance types for the edge compute pool includes:
+
+* `m6i.xlarge`
+* `m5.xlarge`
+* `c5d.2xlarge`
+
+The edge compute pool creates new labels that developers can use to deploy their applications onto those locations. The new labels are:
+
+* `node-role.kubernetes.io/edge=''`
+* `zone_type=local-zone`
+* `zone_group=<LOCAL_ZONE_GROUP>`
+
+
+By default, the system creates the edge compute pool only if users add AWS Local Zone subnet IDs to the list `platform.aws.subnets`.
+
+The edge compute pool's machine sets have a `NoSchedule taint` by default to prevent regular workloads from being spread out on those machines. Users can only run user workloads if the tolerations are defined on the pod spec. This practice is demonstrated in the following examples:
+
+.Example edge pool
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: devcluster.openshift.com
+metadata:
+  name: ipi-localzone
+platform:
+  aws:
+    region: us-west-2
+    subnets:
+    - subnet-0fc845d8e30fdb431
+    - subnet-0a2675b7cbac2e537
+    - subnet-01c0ac400e1920b47
+    - subnet-0fee60966b7a93da6
+    - subnet-002b48c0a91c8c641
+    - subnet-093f00deb44ce81f4
+    - subnet-0f85ae65796e8d107
```

Comment:
Considering this example is related to the CloudFormation exemplified above which creates subnets (public and private) only in 3 AZs: do you think didactic we could replace this with something like:
```
    - subnetID-private1
    - subnetID-private2
    - subnetID-private3
    - subnetID-public1
    - subnetID-public2
    - subnetID-public3
    - subnetID-localzone-public1
```